docs/RELEASE.md

```diff
@@ -79,15 +79,14 @@ The new tag should trigger a new Github release. Verify that it has run by going
 - Source code (zip)
 - Source code (tar.gz)
 
+## Verify the helm chart release
```

Comment:
I realize this means that the helm chart release will exist for x minutes before the image to which it is referring will exist in registries.

This is a problem we had in the past that this change would reintroduce.

I think the benefits outweigh it tho.

Reply:
(still thinking outloud)
Actually let me edit the release process to account for this.